src/Search/Search.js

```diff
@@ -23,7 +74,87 @@ const Search = () => {
     setFilterBy(value);
   };
 
-  return <input onChange={handleChange} value={filterBy} />;
+  const handleClick = () => {
+    setOpen(!open);
+  };
+
+  const handleClickQueryMode = () => {
+    setQueryMode(!queryMode);
+  };
+
+  const openChangeClassName = (show, close) => {
+    return open ? show : close;
+  };
+
+  const createTextField = params => {
+    return (
+      <TextField
+        {...params}
+        id="o2xp-search-bar"
+        color={colorChange}
+        onChange={handleChange}
+        value={filterBy}
+        type="text"
+        placeholder={placeholderQueryMode}
+        className={openChangeClassName("o2xp-open-state", "o2xp-close-state")}
+        error={hasError}
+        helperText={
+          hasError &&
+          queryMode &&
+          "Syntax Error : Check query mode documentation for more informations"
+        }
+      />
+    );
+  };
+
+  return (
+    <div id="o2xp-search-wrap">
+      <div>
+        <div className="o2xp-wrapper-input">
+          {columns.data && queryMode ? (
+            <Autocomplete
+              id="o2xp-autocomplete"
+              className={openChangeClassName("o2xp-open-state", "o2xp-close-state")}
+              freeSolo
+              options={Object.values(columns.data.columns).map(
```

Comment:
AutoComplete is working only with the first parameter ?

Reply:
Yes, I need to find a way to fix it